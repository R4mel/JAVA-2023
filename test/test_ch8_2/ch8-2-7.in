1
17
10
11
12
12
12
0
0
0
