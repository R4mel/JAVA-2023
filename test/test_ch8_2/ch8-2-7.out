
************* Main Menu **************
* 0.exit 1.PersonManager 2.ch2 3.ch3 *
* 4.ch5 5.PMbyMap 6.MyVectorTest     *
**************************************
menu item? 1
[0] p0 10 70.0 false :Gwangju ,Nam-gu , Bongseon-dong 21:
[1] p1 11 61.1 true :Jong-ro 1-gil,Jongno-gu,   Seoul:
[2] s1 1 65.4 true :Jongno-gu Seoul: Physics 3.8 1
[3] s2 2 54.3 false :Yeonje-gu Busan: Electronics 2.5 4
[4] w1 3 33.3 false :Kangnam-gu Seoul: Samsung Director
[5] w2 4 44.4 true :Dobong-gu Kwangju: Hyundai Manager

=================== Person Management Menu =====================
= 0.exit 1.display 2.clear 3.reset 4.remove 5.copy 6.append    =
= 7.insert 8.login 9.dispStudent(5_3) 10.dispPhone(5_3)        =
= 11.find(6_1) 12.wrapper(6_1) 13.shuffle(6_1) 14.setDate(6_1) =
= 15.chAddress(6_2) 16.collections(7_1) 17.fileManager(8_1)    =
================================================================
menu item? 17

====================== File Management Menu =====================
= 0.exit 1.fileList 2.delete 3.rename 4.addFiles 5.addDir       =
= 6.deleteAll 7.show 8.copy 9.append 10.display 11.clear 12.add =
=================================================================
menu item? 10
[0] p0 10 70.0 false :Gwangju ,Nam-gu , Bongseon-dong 21:
[1] p1 11 61.1 true :Jong-ro 1-gil,Jongno-gu,   Seoul:
[2] s1 1 65.4 true :Jongno-gu Seoul: Physics 3.8 1
[3] s2 2 54.3 false :Yeonje-gu Busan: Electronics 2.5 4
[4] w1 3 33.3 false :Kangnam-gu Seoul: Samsung Director
[5] w2 4 44.4 true :Dobong-gu Kwangju: Hyundai Manager
-----------------
6 entries

====================== File Management Menu =====================
= 0.exit 1.fileList 2.delete 3.rename 4.addFiles 5.addDir       =
= 6.deleteAll 7.show 8.copy 9.append 10.display 11.clear 12.add =
=================================================================
menu item? 11
-----------------
0 entries

====================== File Management Menu =====================
= 0.exit 1.fileList 2.delete 3.rename 4.addFiles 5.addDir       =
= 6.deleteAll 7.show 8.copy 9.append 10.display 11.clear 12.add =
=================================================================
menu item? 12
[0] Kimm10 1948 54.4 false :Incheon Dong-gu: Kia CFO
[1] Park11 1095 98.9 true :Seoul Nam-gu: Computer 1.2 1
[2] Choi12 1882 72.8 false :Seoul Jung-gu: Computer 1.1 1
[3] Moon13 1960 81.2 false :Seoul Dong-gu:
[4] Nahh14 1646 93.9 false :Daejeon Buk-gu:
[5] Jooh15 1451 61.5 true :Daejeon Dong-gu:
[6] Moon16 1220 98.2 false :Gwangju Seo-gu:
[7] Kimm17 1668 64.8 false :Seoul Buk-gu: Samsung CFO
[8] Kang18 1067 40.8 true :Gwangju Dong-gu:
[9] Leem19 1290 48.6 false :Gwangju Jung-gu:
-----------------
10 entries

====================== File Management Menu =====================
= 0.exit 1.fileList 2.delete 3.rename 4.addFiles 5.addDir       =
= 6.deleteAll 7.show 8.copy 9.append 10.display 11.clear 12.add =
=================================================================
menu item? 12
[0] Kimm10 1948 54.4 false :Incheon Dong-gu: Kia CFO
[1] Park11 1095 98.9 true :Seoul Nam-gu: Computer 1.2 1
[2] Choi12 1882 72.8 false :Seoul Jung-gu: Computer 1.1 1
[3] Moon13 1960 81.2 false :Seoul Dong-gu:
[4] Nahh14 1646 93.9 false :Daejeon Buk-gu:
[5] Jooh15 1451 61.5 true :Daejeon Dong-gu:
[6] Moon16 1220 98.2 false :Gwangju Seo-gu:
[7] Kimm17 1668 64.8 false :Seoul Buk-gu: Samsung CFO
[8] Kang18 1067 40.8 true :Gwangju Dong-gu:
[9] Leem19 1290 48.6 false :Gwangju Jung-gu:
[10] Yang20 1753 91.1 true :Incheon Jung-gu: Kia CFO
[11] Choi21 1577 40.1 true :Gwangju Jung-gu:
[12] Nahh22 1958 55.3 false :Incheon Jung-gu: Samsung Chief
[13] Shin23 1098 95.5 false :Busan Buk-gu:
[14] Song24 1901 92.4 true :Daegu Buk-gu: Electronics 0.6 4
[15] Park25 1380 69.4 false :Busan Jung-gu: Physics 2.1 2
[16] Johh26 1362 82.0 false :Seoul Dong-gu: Biology 2.1 3
[17] Kimm27 1317 74.8 true :Daejeon Jung-gu:
[18] Shim28 1113 64.5 true :Incheon Jung-gu: Computer 1.5 3
[19] Jooh29 1050 42.4 false :Busan Seo-gu:
-----------------
20 entries

====================== File Management Menu =====================
= 0.exit 1.fileList 2.delete 3.rename 4.addFiles 5.addDir       =
= 6.deleteAll 7.show 8.copy 9.append 10.display 11.clear 12.add =
=================================================================
menu item? 12
[0] Kimm10 1948 54.4 false :Incheon Dong-gu: Kia CFO
[1] Park11 1095 98.9 true :Seoul Nam-gu: Computer 1.2 1
[2] Choi12 1882 72.8 false :Seoul Jung-gu: Computer 1.1 1
[3] Moon13 1960 81.2 false :Seoul Dong-gu:
[4] Nahh14 1646 93.9 false :Daejeon Buk-gu:
[5] Jooh15 1451 61.5 true :Daejeon Dong-gu:
[6] Moon16 1220 98.2 false :Gwangju Seo-gu:
[7] Kimm17 1668 64.8 false :Seoul Buk-gu: Samsung CFO
[8] Kang18 1067 40.8 true :Gwangju Dong-gu:
[9] Leem19 1290 48.6 false :Gwangju Jung-gu:
[10] Yang20 1753 91.1 true :Incheon Jung-gu: Kia CFO
[11] Choi21 1577 40.1 true :Gwangju Jung-gu:
[12] Nahh22 1958 55.3 false :Incheon Jung-gu: Samsung Chief
[13] Shin23 1098 95.5 false :Busan Buk-gu:
[14] Song24 1901 92.4 true :Daegu Buk-gu: Electronics 0.6 4
[15] Park25 1380 69.4 false :Busan Jung-gu: Physics 2.1 2
[16] Johh26 1362 82.0 false :Seoul Dong-gu: Biology 2.1 3
[17] Kimm27 1317 74.8 true :Daejeon Jung-gu:
[18] Shim28 1113 64.5 true :Incheon Jung-gu: Computer 1.5 3
[19] Jooh29 1050 42.4 false :Busan Seo-gu:
[20] Kang30 1324 77.8 false :Seoul Dong-gu: Kia CEO
[21] Shin31 1771 64.6 true :Incheon Buk-gu: Biology 4.1 1
[22] Ryoo32 1780 46.8 false :Gwangju Nam-gu: Computer 3.6 2
[23] Park33 1615 77.2 true :Daegu Nam-gu: Chemistry 3.3 1
[24] Hong34 1618 57.8 false :Gwangju Buk-gu: KT Chief
[25] Jeon35 1471 84.1 true :Daejeon Jung-gu: Chemistry 0.7 1
[26] Seoh36 1384 74.5 false :Daegu Dong-gu: KT CFO
[27] Nahh37 1298 62.7 false :Gwangju Seo-gu:
[28] Baek38 1076 48.4 false :Seoul Seo-gu: Naver CTO
[29] Shin39 1614 54.9 false :Ulsan Nam-gu: Chemistry 1.9 4
-----------------
30 entries

====================== File Management Menu =====================
= 0.exit 1.fileList 2.delete 3.rename 4.addFiles 5.addDir       =
= 6.deleteAll 7.show 8.copy 9.append 10.display 11.clear 12.add =
=================================================================
menu item? 0

=================== Person Management Menu =====================
= 0.exit 1.display 2.clear 3.reset 4.remove 5.copy 6.append    =
= 7.insert 8.login 9.dispStudent(5_3) 10.dispPhone(5_3)        =
= 11.find(6_1) 12.wrapper(6_1) 13.shuffle(6_1) 14.setDate(6_1) =
= 15.chAddress(6_2) 16.collections(7_1) 17.fileManager(8_1)    =
================================================================
menu item? 0

************* Main Menu **************
* 0.exit 1.PersonManager 2.ch2 3.ch3 *
* 4.ch5 5.PMbyMap 6.MyVectorTest     *
**************************************
menu item? 0

Good bye!!
